internal/oidc/oidc.go

```diff
@@ -29,9 +31,25 @@ func PinnipedCLIOIDCClient() *fosite.DefaultOpenIDConnectClient {
 	}
 }
 
-func FositeOauth2Helper(oauthStore interface{}, hmacSecretOfLengthAtLeast32 []byte) fosite.OAuth2Provider {
+func FositeOauth2Helper(issuerURL string, oauthStore fosite.Storage, hmacSecretOfLengthAtLeast32 []byte) fosite.OAuth2Provider {
 	oauthConfig := &compose.Config{
-		EnforcePKCEForPublicClients: true,
+		AuthorizeCodeLifespan: 3 * time.Minute, // seems more than long enough to exchange a code
+
+		IDTokenLifespan:     5 * time.Minute, // match clientCertificateTTL since it has similar properties to this token
+		AccessTokenLifespan: 5 * time.Minute, // match clientCertificateTTL since it has similar properties to this token
```

Comment:
Chatted with Mo and he thought that any value 15 minutes or smaller would be short enough to avoid needing a revocation feature. We should think more about this and decide if 5 minutes is too short.